hoomd/md/nlist.py

```diff
@@ -119,10 +124,12 @@ class Cell(_NList):
 
     Args:
         buffer (float):  Buffer width.
-        exclusions (tuple): ...
+        exclusions (tuple): Excludes pairs from the neighbor list, which 
```

Comment:
We 
```suggestion
        exclusions (tuple[str]): Excludes pairs from the neighbor list, which 
```
In an offline conversation, @joaander and I have decided that we will keep the `[]` notation, but switch to using the standard Python type names.